index.html

```diff
@@ -58,25 +58,25 @@ <h1 class="section-title">
         <ul class="social-media">
```

Comment:
- [ ] [OPTIONAL] In your  <title>Portfolio</title>, change it to a more meaningful title, such as  <title>Polina's Portfolio</title> or  <title>Polina Full Stack Developer</title>

Reply:
Thank you a lot for the suggestion! I will change it :)